src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/EditorTests.java

```diff
@@ -172,7 +172,7 @@ public void MercuryCuratedEditorTest_002_addAndSaveSection() {
     Assertion.assertNumber(
         curatedContent.getCuratedContentItemsNumber(),
         1,
-        "New section items"
+        "If error says that 3 elements were found - it means getList API returned cached response - ticket created: XW-1281"
```

Comment:
Yes it is seen only if assertion failed.
I don't like the word probably - if I add probably then the person who checks the result is going to spend the time to investigating this issue. My intention here was to save this person time